vault-harvester/scripts/deploy.ts

```diff
@@ -58,17 +58,40 @@ const deployContract = async (
   contractFactory: ContractFactory,
   constructorArgs: any[]
 ): Promise<{ contract: string; impl: string }> => {
+  if (network.name !== "matic") {
+    console.log("Deploying: MockKeeperRegistry");
+    const MockKeeperRegistryFactory = await ethers.getContractFactory(
+      "MockKeeperRegistry"
+    );
+    const mockKeeperRegistry = await MockKeeperRegistryFactory.deploy();
+    await mockKeeperRegistry.deployed();
+    console.log(`MockKeeperRegistry: ${mockKeeperRegistry.address}`);
+    constructorArgs[0] = mockKeeperRegistry.address;
+  }
+
   console.log("Deploying:", name);
   const ret = {
     contract: "",
     impl: "",
   };
+
   const deployTx = await contractFactory.deploy(...constructorArgs);
   await deployTx.deployed();
   console.log(`${name}: ${deployTx.address}`);
   const contract = await ethers.getContractAt(name, deployTx.address);
   ret.contract = contract.address;
 
+  if (network.name !== "hardhat" && network.name !== "localhost") {
+    console.log("Waiting to verify...");
+    await deployTx.deployTransaction.wait(6);
+    console.log(`Verifying ${config.harvester.contractName}...`);
+    await run("verify:verify", {
+      address: deployTx.address,
+      contract: `contracts/test/MockHarvester.sol:${config.harvester.contractName}`,
```

Comment:
Why `contracts/test/MockHarvester.sol`? Isn't this part supposed to be not for deploying test contracts?

Reply:
What else you suggest can be deployed in this project?